src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties.component.ts

```diff
@@ -477,6 +483,28 @@ export class CsvPropertiesComponent implements OnInit, AfterViewInit, OnDestroy
         this.temporalProperties.updateValueAndValidity();
     }
 
+    temporalUpdate() {
+        let usedColumns = this.spatialProperties.controls['isWkt'].value ? 1 : 2;
```

Comment:
Maybe it is requiredColumns? I guess there could be used more than 2.
```suggestion
        let requiredColumns = this.spatialProperties.controls['isWkt'].value ? 1 : 2;
```